services/logs-db/opendistro_security/roles_mapping.yml

```diff
@@ -10,6 +10,7 @@ all_access:
   backend_roles:
     - admin
     - platform-owner
+    - platform-viewer
```

Comment:
wondering whether the `readall_and_monitor` role may be a better fit here?
https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/resources/static_config/static_roles.yml

Reply:
Oh, mayhaps.

Reply:
Turns out that role is no good, few things that it has no permissions to do which it needs.
The only role that really fits is all_access without going down the path of creating a custom role.

Reply:
follow-up shows that the `read` role doesn't grant access to all /get functions on indices - particularly aliases :sob: 
https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/resources/static_config/static_action_groups.yml#L83